### PR TITLE
[bitnami/postgresql-ha] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/postgresql-ha/templates/postgresql/witness-statefulset.yaml [hunk 1]
```diff
@@ -55,8 +48,8 @@ spec:
       affinity: {{- include "common.tplvalues.render" (dict "value" .Values.witness.affinity "context" $) | nindent 8 }}
       {{- else }}
       affinity:
-        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.witness.podAffinityPreset "component" "postgresql" "context" $) | nindent 10 }}
-        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.witness.podAntiAffinityPreset "component" "" "context" $) | nindent 10 }}
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.witness.podAffinityPreset "component" "postgresql-witness" "customLabels" $podLabels "context" $) | nindent 10 }}
```
Comment (fevisera) [inline bitnami/postgresql-ha/templates/postgresql/witness-statefulset.yaml:51]: Why are you changing `postgresql` to `postgresql-witness`?

Comment (juan131) [reply bitnami/postgresql-ha/templates/postgresql/witness-statefulset.yaml]: fixed at https://github.com/bitnami/charts/pull/18410/commits/0a09ba9f9564b2336a69c375ce06d66987b0260a

Comment (fevisera) [reply bitnami/postgresql-ha/templates/postgresql/witness-statefulset.yaml]: Thanks!

**Review by fevisera — APPROVED** [review]
LGTM!

**Pull request merged**